core/lib/types/src/tx/execute.rs

```diff
@@ -4,30 +4,53 @@ use zksync_utils::ZeroPrefixHexSerde;
 
 use crate::{ethabi, Address, EIP712TypedStructure, StructBuilder, H256, U256};
 
-/// `Execute` transaction executes a previously deployed smart contract in the L2 rollup.
-#[derive(Clone, Default, Serialize, Deserialize, PartialEq)]
+#[derive(Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
+struct ExecuteSerde {
```

Comment:
A comment on why do we need this structure (and why we don't just slap `derive` on `Execute`) would be helpful

Reply:
done